scripts/tests

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+
+import platform
+import sys
+import os
+import subprocess
+import logging
+import click
+import hashlib
+from functools import reduce
+from enum import Enum
+
+
+def config_logger(verbose):
+    if verbose:
+        logging.basicConfig(format="%(levelname)-5s > %(message)s", level=logging.DEBUG)
+    else:
+        logging.basicConfig(format="%(levelname)-5s > %(message)s", level=logging.INFO)
+
+
+def sha256sum(result):
+    m = hashlib.sha256()
+    m.update(result)
+    return m.hexdigest()
+
+
+def base_run(bin, verbose):
+    if platform.system() == "Linux" and platform.machine() == "x86_64":
```

Comment:
@potsrevennil The qemu-based testing is supposed to work even on AArch64 systems, right?

Can you perhaps add a `--force-qemu` or similar argument to `tests` so we can test with qemu on those systems, too?

Reply:
I expect qemu-based testing to work on `Linux AArch64` systems.
 
They will not work on `macOS` at least for now, as `QEMU` doesn't provide `qemu-aarch64` binary for `macOS`. There is `qemu-system-aarch64` available on macOS, but I don't whether macOS can utilize this to emulate or not.

Therefore, I'll make the `--force-qemu` flag be blocked on `macOS` for now.

Reply:
Ok, thanks for the clarification!